13/umbraco-forms/release-notes.md

```diff
@@ -17,6 +17,25 @@ If you are upgrading to a new major version, you can find information about the
 
 This section contains the release notes for Umbraco Forms 13 including all changes for this version.
 
+#### [**13.1.0**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.1.0) **(March 5th 2024)**
+
+With the introduction of webhooks with Umbraco 13, we've made an update to Forms to allow workflow execution events to trigger webhooks. Workflows are operations that you can associated with form submission, approval or rejections actions. You can use these where you need to notify external systems of the success or failure of a workflow. Read more on [Umbraco Forms wehbooks](./developer/webhooks.md).
+
+We've added an update that will help customers using Forms within locked down production environments.
```

Comment:
This seems like an incomplete sentence or some words are missing from it 🤔

Reply:
Not really, it's just calling out one of the features in the list below.  It's quite specific, so didn't want to go into detail.